tasks/main.yml

```diff
@@ -14,6 +14,12 @@
     home: /srv/{{ dashboards_user }}
     git_checkout: /srv/{{ dashboards_user }}/git
 
+# CRB is needed for some of the required packages
+- name: Enable CRB repo on CentOS9
```

Comment:
Adding a repo with yum_repository requires to add the baseurl, metalink or mirrorlist:

[{"changed": false, "msg": "Parameter 'baseurl', 'metalink' or 'mirrorlist' is required."}](fatal: [localhost]: FAILED! => {"changed": false, "msg": "Parameter 'baseurl', 'metalink' or 'mirrorlist' is required."})

However, I'd say using ansible.builtin.command instead of shell is good. I'll update it.